README.md

```diff
@@ -52,6 +52,10 @@ See **Other examples** below for transactable modes.
 * `WEB3_PROVIDER_URI` - HTTP or WS URL of web3 Ethereum node (tested with Geth). **Required**.
 * `BEACON_NODE` - HTTP endpoint of Beacon Node (Lighthouse recommended, also tested with Prysm). **Required**.
 * `POOL_CONTRACT` - Lido contract in EIP-55 (mixed-case) hex format. **Required**. Example: `0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84`
+* `STABLE_SWAP_POOL_CONTRACT` - address of Curve ETH/stETH stable swap pool. **Required**.
```

Comment:
Let's change the `STABLE_SWAP_` prefix to `STETH_PRICE_ORACLE_`.

Reply:
Just to clarify: `STABLE_SWAP_POOL_CONTRACT` -> `STETH_PRICE_ORACLE_POOL_CONTRACT`, `STABLE_SWAP_STATE_ORACLE_CONTRACT` -> `STETH_PRICE_ORACLE_CONTRACT` correct? And do you mean only prefixes for env variables or for all variable inside `oracle.py` file?

Reply:
Let's go with `STABLE_SWAP_POOL_CONTRACT` -> `STETH_CURVE_POOL_CONTRACT`, `STABLE_SWAP_STATE_ORACLE_CONTRACT` -> `STETH_PRICE_ORACLE_CONTRACT`. I mean both env vars and python vars.